server/test/com/cloud/user/AccountManagerImplTest.java

```diff
@@ -231,6 +253,73 @@ public void cleanup() {
         CallContext.unregister();
     }
 
+    public UsageEventUtils setupUsageUtils() {
+        _usageEventDao = new MockUsageEventDao();
+        UsageEventUtils utils = new UsageEventUtils();
+
+        Map<String, String> usageUtilsFields = new HashMap<String, String>();
+        usageUtilsFields.put("usageEventDao", "_usageEventDao");
+        usageUtilsFields.put("accountDao", "_accountDao");
+        usageUtilsFields.put("dcDao", "_dcDao");
+        usageUtilsFields.put("configDao", "_configDao");
```

Comment:
Why not declare this map as a static constant?  It would shorten this method and make it easier to understand.   Also, consider using the ``ImmutableMap`` type for the type of the constant.